llvm/lib/Target/AMDGPU/SIFoldOperands.cpp

```diff
@@ -861,7 +903,6 @@ bool SIFoldOperandsImpl::tryToFoldACImm(
 
   // Maybe it is just a COPY of an immediate itself.
   MachineInstr *Def = MRI->getVRegDef(UseReg);
-  MachineOperand &UseOp = UseMI->getOperand(UseOpIdx);
   if (!UseOp.getSubReg() && Def && TII->isFoldableCopy(*Def)) {
```

Comment:
Is this now a duplicate check from the one above? Or is the check above temporary until the TODOs have been dealt with?